includes/integration.php

```diff
@@ -14,7 +14,7 @@ class Metrilo_Woo_Analytics_Integration extends WC_Integration {
     private $orders_per_import_chunk = 25;
     private $recent_orders_sync_days = 7;
     private $batch_calls_queue = array();
-    private $possible_events = array('view_product' => 'View Product', 'view_category' => 'View Category', 'view_article' => 'View Article', 'add_to_cart' => 'Add to cart', 'remove_from_cart' => 'Remove from cart', 'view_cart' => 'View Cart', 'checkout_start' => 'Started Checkout', 'identify' => 'Identify calls');
+    private $possible_events = array('view_product' => 'View Product', 'view_category' => 'View Category', 'view_article' => 'View Article', 'add_to_cart' => 'Add to cart', 'remove_from_cart' => 'Remove from cart', 'view_cart' => 'View Cart', 'checkout_start' => 'Started Checkout', 'identify' => 'Identify calls', 'pageview' => 'Page View');
```

Comment:
Okay, lets use the chance to write consistent titles.
```php
'view_product' => 'Viewed A Product',
'view_category' => 'Viewed A Category',
'view_article' => 'Viewed An Article',
'add_to_cart' => 'Added To The Cart',
'remove_from_cart' => 'Removed From The Cart',
'view_cart' => 'Viewed The Cart',
'checkout_start' => 'Started Checkout',
'identify' => 'Identified Oneself',
'pageview' => 'Viewed A Page'
```
@vgochev What do you reckon?